src/lib/authserver/dir/client.rb

```diff
@@ -23,6 +25,7 @@ def initialize(url, bind_dn, bind_pw)
 
   # modify invokes ldapmodify and returns tuple of command output and boolean (success or not).
   def modify(ldif_input, ignore_existing)
+    log.info('modify: %s' % ldif_input)
```

Comment:
just hint. Format ( aliases by %operator) is mainly useful for translations. For logging usually interpolation is easier. So it can look like `log.info "modify: #{ldif_input}"`